kite-data/kite-data-core/src/test/java/org/kitesdk/data/spi/TestRefinableViews.java

```diff
@@ -819,4 +820,27 @@ public void testNotPartitioned() throws Exception {
     Assert.assertNotNull("with should succeed",
         notPartitioned.with("timestamp", now));
   }
+
+  @Test
+  public void testReadySignals() {
```

Comment:
Added some additional tests cover the different RefinableView possibilities 9d3ba86